manticore/core/workspace.py

```diff
@@ -94,7 +94,7 @@ def load_value(self, key):
             return s.read()
 
     @contextmanager
-    def save_stream(self, key):
+    def save_stream(self, key, *rest, **kwargs):
```

Comment:
Can we only leave `*rest, **kwargs` and have `key = rest[0]` or it's not clean enough?